node/connection_base.js

```diff
@@ -99,6 +144,14 @@ TChannelConnectionBase.prototype.handleCallRequest = function handleCallRequest(
     req.remoteAddr = self.remoteName;
     self.ops.addInReq(req);
 
+    if (self.draining && (
```

Comment:
This is in the wrong place ? We add the inReq and then check draining.

Let's avoid adding the InReq.